nemo/collections/tts/modules/common.py

```diff
@@ -185,10 +158,11 @@ def __init__(
             self.dense = nn.Linear(n_channels, out_dim)
 
     def forward(self, context: Tensor, lens: Tensor) -> Tensor:
-        mask = get_mask_from_lengths_and_val(lens, context)
+        mask = get_mask_from_lengths(lens, context.shape[-1])
         mask = mask.to(dtype=context.dtype).unsqueeze(1)
         for conv in self.convolutions:
             context = self.dropout(F.relu(conv(context, mask)))
+        context = context * mask
```

Comment:
Will this affect pretrained models ?

Reply:
This was in fact not needed - context is immediately passed to pack sequence call